example/provider-local/garden/base/secretbinding.yaml

```diff
@@ -12,6 +12,8 @@ kind: SecretBinding
 metadata:
   name: local
   namespace: garden
+  labels:
+    cloudprofile.garden.sapcloud.io/name: local
```

Comment:
I am not sure we have to do it. Ref https://github.com/gardener/dashboard/issues/781

SecretBinding/CredentialsBinding already have a `provider.type` field. I hope this is enough for the dashboard to determine the provider type of the Secret. 

cc @grolu @petersutter

Reply:
That's a pretty old issue. 😃
/close

Reply:
Yes, I know that it is an old issue. But wouldn't you agree that it is conceptually wrong the Gardener dashboard to define and impose such a contract to Gardener itself?

@grolu @petersutter https://github.com/gardener/dashboard/issues/781 is striking again and again over the years isn't ok to rely on the `.provider.type` field of a SecretBinding/CredentialsBinding? Or do you want to know the CloudProfile name because of the per domain CloudProfiles on openstack?

Reply:
> But wouldn't you agree that it is conceptually wrong the Gardener dashboard to define and impose such a contract to Gardener itself?

Yes, And it should be fixed as you mentioned. That's why I closed the PR.

Reply:
@ialidzhikov 
>@grolu @petersutter https://github.com/gardener/dashboard/issues/781 is striking again and again over the years. Isn't ok to rely on the .provider.type field of a SecretBinding/CredentialsBinding? Or do you want to know the CloudProfile name because of the per domain CloudProfiles on openstack?

This is because of how we implemented secrets in the Dashboard. Right now, the selected cloud profile determines which secret(s) we show on the create cluster page. This makes it easier for the user as some infrastructure providers (e.g. OpenStack) can have multiple underlying landscapes an secrets work only on a specific one.
However, we could get rid of the label and only rely on the type field. Regarding namespaced cloud profiles this may makes sense anyway. This is something I cannot decide as I don't know how it would impact usability.